docs/standaard/documenten/index.md

```diff
@@ -293,6 +293,18 @@ Indien een verzoek één of meer expand parameters bevat MOET het attribuut onde
 Op een verzoek MOET een geldige response zoals deze opgevraagd is opleveren. Indien een verzoek één of meer expand parameters bevat MOET ook de te expanderen informatie opgehaald en teruggegeven kunnen worden. Indien geen geldige response kan worden teruggegeven moet een foutmelding (http 404) worden teruggegeven.
 
 
+Het expand patroon staat beschreven op [deze pagina](../expand_patroon). In onderstaande tabel staat beschreven welke velden in de Documenten API geëxpandeerd kunnen worden. Indirect kunnen vanuit de Zaken API verwijzingen naar de Catalogi API worden ontsloten. Een volledig overzicht van velden die geëxpandeerd kunnen worden staat op [deze pagina](../expand_patroon)
+
+| API        | Versie vanaf | Resource                      | Veld met url           | Expand   | Doel API   | Doel resource                 | Opmerkingen                             |
+| ---------- | ------------ | ----------------------------- | ---------------------- | -------- | ---------- | ----------------------------- | --------------------------------------- |
+| documenten | 1.4.0        | enkelvoudiginformatieobjecten | informatieobjecttype   | Ja       | catalogi   | informatieobjecttypen         |                                         |
+| documenten | 1.4.0        | gebruiksrechten               | informatieobject       | Ja       | documenten | enkelvoudiginformatieobjecten |                                         |
+| documenten | 1.4.0        | verzendingen                  | informatieobject       | Ja       | documenten | enkelvoudiginformatieobjecten |                                         |
+| catalogi   | 1.3.0        | informatieobjecttypen         | catalogus              | Indirect | catalogi   | catalogussen                  |                                         |
+| catalogi   | 1.3.0        | informatieobjecttypen         | zaaktypen              | Indirect | catalogi   | zaaktypen                     |                                         |
+| catalogi   | 1.3.0        | informatieobjecttypen         | besluittypen           | Indirect | catalogi   | besluittypen                  |                                         |
+
```

Comment:
Ik zou de laatste drie rijen uit de tabel weglaten. Immers je kunt via geneste expand veel meer resources uit de Catalogi API op indirecte wijze expanden en het gaat wat te ver om die hier ook allemaal op te nemen.

Reply:
> Immers je kunt ook in de Documenten API via geneste expand alle resources van de Catalogi API bereiken.


Vanuit de Documenten API kun je alleen de resources van de Catalogi API benaderen waar de resources van de Documenten API ook daadwerkelijk naar verwijzen.  Een ZaaktypeInformatieobjecttype bv. wordt niet gebruikt in de Documenten API en kan dus van daaruit ook niet geëxpandeerd worden. Hetzelfde geldt voor een zaaktype. Dat zijn gegevens die alleen vanuit de Zaken API benaderd kunnen worden.

Ook indirect kunnen deze gegevens niet benaderd worden, er is immers geen relatie vanuit EnkelvoudigInformatieObjecttype naar ZaaktypeInformatieobjecttype. Die relatie ligt andersom, vanuit het ZaaktypeInformatieobjecttype naar het Enkelvoudiginformatieobjecttype.

Zo gezien is het beperken van de vanuit de Documenten API te bereiken gegevens in de Catalogi API dus correct mijns inziens.  Hetzelfde geldt voor de Besluiten API.

Reply:
> Een ZaaktypeInformatieobjecttype bv. wordt niet gebruikt in de Documenten API en kan dus van daaruit ook niet geëxpandeerd worden

Ja dat klopt, momenteel kan de resource `zaaktypeinformatieobjecttypen` niet bereikt worden via geneste expands. Dat is volgens mij een fout in de OAS en ook een fout in de excel-spreadsheet die nog hersteld moet worden.

> Hetzelfde geldt voor een zaaktype

Deze opmerking begrijp ik niet. Immers de "zaaktypen" die bij een informatieobjecttype  kunnen wel genest verder geëxpandeerd worden. Bijvoorbeeld:

GET  /enkelvoudiginformatieobjecten?expand=informatieobjecttypen.zaaktypen.roltypen

Dat houdt in dat alle rijen met de resource "zaaktypen" ook opgenomen moeten worden, etc.

Reply:
> Dat is volgens mij een fout in de OAS en ook een fout in de excel-spreadsheet die nog hersteld moet worden.

![image](https://github.com/VNG-Realisatie/gemma-zaken/assets/37145898/bd653b56-4ff7-4795-89d1-90126ca35470)

Het attribuut "informatieobjecttypen" is niet van het type URI en heeft bovendien geen beschrijving. Volgens mij zou dit een verwijzing moeten zijn naar de resource `zaaktypeinformatieobjecttypen` en dan kan deze resource wel bereikt worden via nesting in de expand.